Apsitvarkom.Api/Controllers/PollutedLocationController.cs

```diff
@@ -131,4 +134,31 @@ public async Task<ActionResult<PollutedLocationResponse>> Create(PollutedLocatio
 
         return CreatedAtAction(nameof(GetById), new { response.Id }, response);
     }
-}
\ No newline at end of file
+
+    [ProducesResponseType(StatusCodes.Status201Created)]
```

Comment:
When looking into the [status codes](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_success), 201 is known to be used when a new resource is created. Here we only edit an already existent resource -> we don't create a new one.

A [common practice](https://www.moesif.com/blog/technical/api-design/Which-HTTP-Status-Code-To-Use-For-Every-CRUD-App/#update) is using 200 (OK) or 204 (NoContent). As you seem to have implemented it as an `Ok` (line 157), let's keep it at 
```suggestion
    [ProducesResponseType(StatusCodes.Status200OK)]
```